### ISSUE TITLE
Support for Contour Plots

### ISSUE THREAD

**Issue by asewnath** [danholdaway/eva#208]
## Description

@Dooruk and I recently had a discussion about adding contour plot support to eva. I was wondering if anyone at EMC has been looking into this or has any thoughts about it before we begin development. 

@CoryMartin-NOAA @ADCollard @kevindougherty-noaa @EdwardSafford-NOAA

**Comment by Dooruk** [comment]
I would like to create ocean vertical transect plots with EVA. I think I can mess with reducing dimensions for plotting with the EVA transform. However I'm not sure how complicated adding the contour plot would be. Would we have to include contour plot class from `emcpy` to here:

`src/eva/plotting/batch/emcpy/diagnostics`

**Comment by Dooruk** [comment]
@CoryMartin-NOAA I would like to revisit this issue after our brief chat. We do use map plots for 3d plotting but don't we also need the `contourf` included in EVA plotting base class from `emcpy` to for transect plots?

**Comment by CoryMartin-NOAA** [comment]
Yes @Dooruk I think we do need that, and also probably unfilled contours. Do you have anyone to help work on this? If not, let me see if I can get someone at EMC to take the lead on it

**Comment by Dooruk** [comment]
> Yes [@Dooruk](https://github.com/Dooruk) I think we do need that, and also probably unfilled contours. Do you have anyone to help work on this? If not, let me see if I can get someone at EMC to take the lead on it

No unfortunately, the new person will start in a few weeks. If you can assign someone that would be great.

**Comment by CoryMartin-NOAA** [comment]
Ok let me see what I can do
